src/Microsoft.Diagnostics.Monitoring.WebApi/Models/CallStackResults.cs

```diff
@@ -11,12 +12,18 @@ public class CallStackFrame
         [JsonPropertyName("methodName")]
         public string MethodName { get; set; }
 
+        [JsonPropertyName("methodToken")]
```

Comment:
I'm a bit surprised this isn't causing a test failure. This should be changing the api surface and require a regeneration of our `openapi.json` file

Reply:
Spoke offline - there are missing schemas in the json (like callstackframe) that will need to be addressed outside this PR.